locales/bg/bg.json

```diff
@@ -626,16 +626,16 @@
     "Token Name": "Име на символ",
     "Tonga": "Тонга",
     "Too Many Requests": "Твърде много заявки",
-    "total": "ваш",
-    "Total:": "Total:",
+    "total": "обща сума",
+    "Total:": "Обща сума:",
     "Trashed": "Победен",
     "Trinidad And Tobago": "Тринидад и Тобаго",
-    "Trinidad and Tobago": "Trinidad and Tobago",
+    "Trinidad and Tobago": "Република Тринидад и Тобаго",
```

Comment:
Why is it different with previous tag ?

Reply:
The second means "Republic of Trinidad and Tobago", the official naming.

Reply:
This is a feature of Laravel projects - they don't always use the same spelling. Therefore, we indicate all the keys for transfers.
I recommend using the same translation for both keys.

Reply:
Updated accordingly.